.github/issue_template.md

```diff
@@ -1,3 +1,9 @@
+Before opening a new issue, please double check the passed issues (both open and
+closed ones) so see if your problem has been discussed before and already
+contains an answer/solution. Likewise, check the FAQ in the Documentation folder
```

Comment:
Even though this will show up as editable text and not a clickable link, I wonder if a URL to the page that the user can quickly copy-and-paste would make this easier/more likely to be read.

Reply:
added.